__tests__/Auth0Client.test.ts

```diff
@@ -226,6 +587,16 @@ describe('Auth0Client', () => {
         auth0Client
       );
     });
+
+    it('should throw an error on token failure', async () => {
+      const auth0 = setup();
+
+      await expect(
+        loginWithRedirect(auth0, false, {}, TEST_CODE, 'MTIz')
+      ).rejects.toThrowError(
+        'HTTP error. Unable to fetch https://auth0_domain/oauth/token'
+      );
+    });
```

Comment:
I added one test to `loginWithRedirect`, to verify it throws when the token call fails.